checkpoint: "/home/TAP-VINS/catkin_ws/tap/co-tracker/checkpoints/scaled_online.pth"
offline_checkpoint: "/home/TAP-VINS/catkin_ws/tap/co-tracker/checkpoints/scaled_offline.pth"
device: "cuda"
debug: true

local_grid_size: 0 # Size of the local grid, set to 0 for no local grid, set to n for an n x n grid
local_grid_extent: 0 # Patch size